osu.ElasticIndexer/Commands/PumpAllScoresCommand.cs

```diff
@@ -46,15 +47,18 @@ public int OnExecute(CancellationToken cancellationToken)
                 if (cancellationToken.IsCancellationRequested)
                     break;
 
+                List<ScoreItem> scoreItems = new List<ScoreItem>();
+
                 foreach (var score in scores)
                 {
                     score.country_code ??= "XX";
+                    scoreItems.Add(new ScoreItem { Score = score });
+                }
 
-                    if (Verbose)
-                        Console.WriteLine($"Pushing {score}");
+                if (Verbose)
```

Comment:
the distinction between `Verbose` and `!Verbose` below isn't needed anymore if pushing in batches.